doc/releases/mimic.rst

```diff
@@ -388,6 +388,11 @@ These changes occurred between the Luminous and Mimic releases.
     via a "pass through" method. See http://docs.ceph.com/docs/master/mgr/restful
     for details.
 
+  - New command to track throughput and IOPS statistics, also available in
+    `ceph -s` and previously in `ceph -w`. To use this command, enable
+    the `iostat` Manager module and invoke it using `ceph iostat`. See the
+    :ref:`iostat documentation <mgr-iostat-overview>` for details.
```

Comment:
Let's add this to mgr_initial_modules so that it is enabled by default.

Reply:
(And let's adjust the language on the module page to reflect that)

Reply:
...although upgraded clusters won't have it enabled.  Hmm.  Should we enable it automatically during upgrade?  @jcsp